service/src/main/kotlin/app/cash/backfila/client/HttpClientServiceClientProvider.kt

```diff
@@ -11,17 +14,14 @@ import okhttp3.Interceptor
 import retrofit2.Retrofit
 import retrofit2.adapter.guava.GuavaCallAdapterFactory
 import retrofit2.converter.wire.WireConverterFactory
-import javax.inject.Inject
-import javax.inject.Named
-import javax.inject.Singleton
 
 @Singleton
 class HttpClientServiceClientProvider @Inject constructor(
-    @Named(HTTP) private val defaultHttpClientEndpointConfig: HttpClientEndpointConfig,
-    private val httpClientFactory: HttpClientFactory,
-    private val httpClientConfigUrlProvider: HttpClientConfigUrlProvider,
-    @HttpClientNetworkInterceptor private val networkInterceptors: List<Interceptor>,
-    private val moshi: Moshi
+  @Named(HTTP) private val defaultHttpClientEndpointConfig: HttpClientEndpointConfig,
```

Comment:
im confused where 2 space hanging indent is coming from

Reply:
Yeah me too. I'm not seeing it in intellij